pool/static_pool/pool.go

```diff
@@ -201,8 +201,8 @@ begin:
 		rsp, err = w.Exec(context.Background(), p)
 	}
 
-	if sp.cfg.MaxJobs != 0 && w.State().NumExecs() >= sp.cfg.MaxJobs {
-		sp.log.Debug("max requests reached", zap.Int64("pid", w.Pid()))
```

Comment:
`jobs` vs `execs` vs `requests` - only one can survive

Reply:
`requests` should survive 😜